tools/test/test_depsolve.py

```diff
@@ -40,6 +40,23 @@ def assert_dnf():
         raise RuntimeError("Cannot import dnf")
 
 
+def is_license_expression_available():
+    """
+    Check if the license-expression package is available.
+
+    The check is not done by importing the package in the current Python environment, because it may be
```

Comment:
Maybe a tiny addition here like:
```
The check is not done by importing the package in the current Python environment, because "osbuild-depsolve-dnf" is run outside of any virtualenv that that the tests may run in and inside "osbuild-depsolve-dnf" the import for "license_expression" happens.
```
(this might be a bit wordy but I hope it spell the reason out a bit more explicitly)

Reply:
I'll defer this to a follow-up.